src/univariate/continuous/normal.jl

```diff
@@ -1,28 +1,41 @@
 """
-    Normal(μ,σ)
+    Normal <: ContinuousUnivariateDistribution
 
-The *Normal distribution* with mean `μ` and standard deviation `σ≥0` has probability density function
+The *normal* or *Gaussian* probability distribution. 
+
+# Constructors
+
+    Normal(μ|mu|mean=0, σ|sigma|std=1)
+
+Construct a `Normal` distribution object with mean `μ` and standard deviation `σ`,
+
+    Normal(μ|mu|mean=0, σ²|var=1)
```

Comment:
I think it's worth it: in some cases it's more convenient to think in terms of `std`, in some cases `var`.

I'm not sure I understand you point about ambiguity for `Logistic`: the point of this was to make it less ambiguous?

Reply:
> I'm not sure I understand you point about ambiguity for `Logistic`: the point of this was to make it less ambiguous?

Sorry, `Logistic` wasn't a good example since it supports three different meanings for the second argument. Actually `Normal` illustrates my point better: if we allow both `Normal(μ=x, std=y)` and `Normal(μ=x, var=y)`, it's not clear what `Normal(x, y)` means. Then we have two solutions:
- Deprecate positional arguments, as you proposed earlier. That requires adding inner constructors to ensure the default constructors are not generated.
- Drop the `var` methods, so that positional arguments are not ambiguous (at least for distributions that don't have other parameterizations).

Reply:
I don't really see it as a problem, but if others think it is then I would lean toward the first.

Relatedly, I've tried to keep the convention that the first signature in the help matches the positional layout.

Reply:
I also don't see much of a problem, in a similar situation also `range` allows both `range(start; length=n)` and `range(start; stop=n)` and the positional signature is `range(start, stop)`.